graphula-core/src/Graphula.hs

```diff
@@ -16,6 +16,7 @@
   @
 -}
 {-# LANGUAGE ConstraintKinds #-}
+{-# LANGUAGE ConstrainedClassMethods #-}
```

Comment:
Huh, I didn't know this wasn't Haskell98. There's a pretty funny comment about it [here](https://downloads.haskell.org/~ghc/8.0.2/docs/html/users_guide/glasgow_exts.html#class-method-types):
> The restriction is a pretty stupid one in the first place, so -XConstrainedClassMethods is implied by -XMultiParamTypeClasses.